new/property.md

```diff
@@ -0,0 +1,206 @@
+---
+RFC: unassigned
+Author: John Keiser <jkeiser@chef.io>
+Status: Draft
+Type: Standards Track
+---
+
+# Resource Properties
+
+We add `property` DSL to resources, similar to (and interoperable with) LWRP `attribute`.
+
+It works very similarly to attribute syntax.  There are no backwards compatibility issues with this proposal, as all `attribute` functionality remains the same.
+
+## Motivation
+
+    As a Chef user,
+    I want resource attributes and node attributes to use different words,
+    so that they don't lead me to conflate them as concepts.
+
+## Specification
+
+### Defining properties
+
+Resource class definitions may now call `property` to create a resource property.  This works similarly to LWRP attribute, but with some important additions and differences.
+
+```ruby
+class MyResource < Chef::Resource
+  property :path, Path, name_attribute: true
+  property :content, String
+end
+```
+
+This calls MyResource.properties[name] = <property type>.
+
+When a property is defined this way:
+- `my_resource.<name>` will get the property value.
+- `my_resource.<name> = <value>` will set the property value.
+- `my_resource.<name> <value>` will set the property value.
+- `MyResource.state_attrs` will include the property name.
+- `MyResource.properties` will contain the property type.
+- The setter and getter manipulate the class variable `@<name>`.
+
+#### Use `property` instead of `attribute` in documentation
```

Comment:
@jamescott might have an opinion on the feasibility of updating all documentation to correctly distinguish between resource 'property' and node 'attribute'

@burtlo might want to weigh in on updating all training materials and video. Also, whether this will provide more clarity to new users or be a net loss in usability.